java/openjdk-java/portfile

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                openjdk-java
+version             17
+categories          java
+platforms           darwin
+description         an Open Source, Open Innovation, Open Standards implementation of the Java Platform, Standard Edition, and related projects.
+homepage            openjdk.java.net
+license             GPL-2+CE
+universal_variant   no
+
+set long_description_openjdk-ga \
+ Production-ready open-source builds of the Java Development Kit
```

Comment:
The indentation of this and several other lines do not conform to the modeline which specifies that indentations shall be at multiples of 4 spaces.

Reply:
Fixed

Reply:
Changed indentation of all lines to confirm to the modeline which specifies that indentations shall be at multiples of 4 spaces.